app/assets/javascripts/scan_analytics.js.coffee

```diff
@@ -0,0 +1,184 @@
+App.ScanAnalytics =
+  init: ->
+    $('#scan_data_row > .project_row').slick
+      arrows: false
+      speed: 900
+      infinite: true
+      slidesToShow: 1
+      slidesToScroll: 1
+      autoplay: true
+      autoplaySpeed: 10000
+      dots: true
+      adaptiveHeight: true
+    if $('#scan_data').length > 0
+      scanDataFetch()
+      $('#scan_data').on 'ajax:before', ->
+        $('.overlay-loader').show()
+        return
```

Comment:
We do not need explicit `return` statements in coffeescript. You may remove all `return` statements. Also remove extra newlines at line 18.